### PR TITLE
chore(cypress npm scripts): replace scripts with concurrently & wait-on

### PR DESCRIPTION
**Pull request by Mohammer5**
Thanks to @amcgee I was able to get rid of the scripts folder and do everything with concurrently and wait-on

I've also refactored the feature and spec files to match the feature / scenario descriptions

### CODE REVIEW

File: cypress/assets/unfetch.umd.js [hunk 1]
```diff
@@ -1,3 +1,12 @@
+/**
+ * See
+ *
+ * * https://github.com/cypress-io/cypress/issues/95
+ * * https://github.com/cypress-io/cypress-example-recipes/tree/master/examples/stubbing-spying__window-fetch
+ * * https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/stubbing-spying__window-fetch/cypress/integration/polyfill-fetch-from-tests-spec.js
+ *
+ * for an explanation why this is currently necessary...
+ */
```
Comment (varl) [inline cypress/assets/unfetch.umd.js]: Not possible to do something like they did in Maps instead of including a vendor lib?

Comment (Mohammer5) [reply cypress/assets/unfetch.umd.js]: No, the maps app itself does that (see [@austins comment](https://github.com/dhis2/ui-core/pull/513#issuecomment-543113963)), which means that we'd have to do that in the ui-widgets library itself.

Comment (Mohammer5) [reply cypress/assets/unfetch.umd.js]: We could argue about whether it's necessary at all do stub network requests.
There is another possible solution: Create scenario specific storybook scenarios and provide the expected response in the CustomDataProvider.

My preferred solution is the current way, see [my comment](https://github.com/dhis2/ui-core/pull/513#issuecomment-543117723) on the `ui-core`'s cucumber PR

Comment (varl) [reply cypress/assets/unfetch.umd.js]: OK, let's go with your preferred way and try it out.

**Pull request merged**